product_multi_image/models/product_product.py

```diff
@@ -0,0 +1,84 @@
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3).
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _name = "product.product"
+    _inherit = [_name, "base_multi_image.owner"]
+
+    # Make this field computed for getting only the available images
+    image_ids = fields.One2many(
+        comodel_name="base_multi_image.image",
+        compute="_compute_image_ids",
+        inverse="_inverse_image_ids",
+    )
+
+    # image, image_medium, image_small fields are not available since 13.0
```

Comment:
I'm wondering. Could this comment be removed as it has been done in 13.0 and since then other formats are computed from `image_1920`?
https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/models/image_mixin.py#L16